visualization/FairnessDashboard/src/Localization/en.json

```diff
@@ -43,13 +43,13 @@
     "Intro": {
         "welcome": "Welcome to the",
         "fairlearnDashboard": "Fairlearn dashboard",
-        "introBody": "The Fairlearn dashboard enables you to assess tradeoffs between accuracy and fairness of your models",
-        "explanatoryStep": "To set up the assessment, you need to specify a sensitive feature and an accuracy metric.",
+        "introBody": "The Fairlearn dashboard enables you to assess tradeoffs between performance and fairness of your models",
+        "explanatoryStep": "To set up the assessment, you need to specify a sensitive feature and a performance metric.",
         "getStarted": "Get started",
         "features": "Sensitive features",
         "featuresInfo": "Sensitive features are used to split your data into groups. Fairness of your model across these groups is measured by disparity metrics. Disparity metrics quantify how much your model's behavior varies across these groups.",
-        "accuracy": "Accuracy metric",
-        "accuracyInfo": "Accuracy metrics are used to evaluate the overall quality of your model as well as the quality of your model in each group. The difference between the extreme values of accuracy is reported as the disparity in accuracy."
+        "accuracy": "Performance metric",
+        "accuracyInfo": "Performance metrics are used to evaluate the overall quality of your model as well as the quality of your model in each group. The difference between the extreme values of model performance is reported as the disparity in performance."
```

Comment:
A small tweak:
`The difference between the extreme values of the performance metric across the groups is reported as ...`